src/entities/cardCVCNumberInputs/types/CardCVCNumber.types.ts

```
@@ -0,0 +1,4 @@
+import { State, Error } from "../../../shared/types/index.types";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)